bin/vue-init

```diff
@@ -107,7 +107,11 @@ function run () {
           checkDistBranch(template, downloadAndGenerate)
         }
       } else {
-        downloadAndGenerate(template)
+        if (template.indexOf('#') !== -1) {
+          downloadAndGenerate(template)
+        } else {
+          checkDistBranch(template, downloadAndGenerate)
+        }
```

Comment:
I think `vue-init` file shouldn't be changed for this feature. Please merge master into your dev branch and make sure `vue-init` isn't changed.